benchmark/bulk_insert.cpp

```diff
@@ -75,6 +75,47 @@ void print_time_csvline(std::string db_name,
     log_file.close();
 }
 
+void insert_stats_to_session(WT_SESSION *session, int edgeNo, int nodeNo)
+{
+    WT_CURSOR *cursor;
+    if (int ret = session->open_cursor(
+                      session, "table:metadata", NULL, NULL, &cursor) != 0)
+    {
+        std::cout << "Failed to open metadata table";
+    }
+    cursor->set_key(cursor, node_count.c_str());
+    cursor->set_value(cursor, std::to_string(nodeNo).c_str());
```

Comment:
I think we should use ```sprintf``` or ```itoa``` to convert the ```int``` to ```char*```

```itoa``` will only work for 32-bit ints. sprintf works for any size ints as long as the buffer is large enough.

Reply:
How long should I set the buffer to be? Should I create a new constant in common.h?